### ISSUE TITLE
[Accessbility]A11y_Java Tooling for IntelliJ/Eclipse - New Features_Install Details-Sample Scala_Keyboard: After selecting the project type, when we press "Tab" key focus is going to some hidden area

### ISSUE THREAD

**Issue by InduPriya1805** [microsoft/azure-tools-for-java#3588]
GitHub Tags: #A11y_JavaToolingforIntelliJ/Eclipse–NewFeature; #A11yMAS; #A11yTCS; #C+E-H; #DesktopApp; #Win10; #JavatoolingforIntelliJ/Eclipse; #Keyboard; #MAS2.4.3; 

[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier

**Environment Details:**
Download and Install IntelliJ IDEA from https://www.jetbrains.com/idea/download/#section=windows
IntelliJ IDEA 2019.2.1
Build #IU-192.6262.58

**OS Details:**
OS: Windows (Version 1903, Build: 18362.113)

MAS Violated: 2.4.3

**Repro Steps:**
1)Launch IntelliJ IDEA
2)Tab till create new project-->Azure Spark/HDInsight
3)Verify After selecting the project type, when we press Tab focus is going to some hidden area or not

**Actual Result:**
After selecting the project type, when we press Tab focus is going to some hidden area

**Expected Result:**
After selecting the project type, when we press Tab focus should not go to any hidden area. It should go to build tool dropdown.

**User Impact:**
Disabled users will find difficult while working if focus is going to any hidden area

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={0de7fbe1-ad7e-48e5-bcbb-8d986691e2b9}

**Attachment:**
[Bug_3588_taking extra tab to go to build tool control.pptx](https://github.com/microsoft/azure-tools-for-java/files/3641228/Bug_3588_taking.extra.tab.to.go.to.build.tool.control.pptx)

**Comment by t-rufang** [comment]
Note: When focus is at `Spark Project with Samples(Scala)`, press tab, focus go to somewhere invisible. Tab again, focus go to `Maven` combobox.
![image](https://user-images.githubusercontent.com/32627233/65677258-dd523280-e083-11e9-9ef8-78b2f2a5c52a.png)

**Comment by t-rufang** [comment]
After we debug the code, we find the focus order is `JBList` -> `JTextPane` -> `JComboBox`. When the focus is at the `JTextPane`, it's invisible.
![image](https://user-images.githubusercontent.com/32627233/66984170-cd5ec900-f0ec-11e9-96d5-818172c0132f.png)

Here the `JBlist` and the `JTextPane` are both sub-components of `ProjectTemplateList`, which is developed by IntelliJ team.
![image](https://user-images.githubusercontent.com/32627233/66984384-2fb7c980-f0ed-11e9-8154-02aa433feea5.png)

Please find some other simialr situation at `Scala` and `Kotlin` wizard, which are both developed by IntelliJ team. (The below gif is for `Scala` project creation wizard)
![issue_3588](https://user-images.githubusercontent.com/32627233/66985223-e49eb600-f0ee-11e9-8bf6-a3afe2aeea90.gif)

**Comment by Alexandria-R** [comment]
This has been approved to close by SME.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749691
